internal/console/pre_scan.go

```diff
@@ -51,21 +51,15 @@ func preRun(cmd *cobra.Command) error {
 func setupConfigFile() (bool, error) {
 	if flags.GetStrFlag(flags.ConfigFlag) == "" {
 		path := flags.GetMultiStrFlag(flags.PathFlag)
-		if len(path) == 0 {
-			return true, nil
-		}
 		if len(path) > 1 {
 			warnings = append(warnings, "Any kics.config file will be ignored, please use --config if kics.config is wanted")
 			return true, nil
 		}
-		configPath := path[0]
-		info, err := os.Stat(configPath)
+
+		configPath, err := os.Getwd()
```

Comment:
Hello, @lipeavelar!

So nice to see you here ❤️We miss you a lot! I hope you are doing well!

Since this can be considered a possible breaking change, can you verify both approaches and document them, please?

Reply:
Hi, @rafaela-soares 🙂
miss you guys too :( I hope you all are doing well!! ❤️
I will document this change better ASAP, I think at the start of next week I get this done. 😉

Reply:
@rafaela-soares, I was rechecking this PR, I think the necessary update on docs is already done here: docs/configuration-file.md

Also, I think, in most cases, this is not a breaking change, since the way KICS works will be almost the same, this changes will only affect people who execute KICS from other folder than where config file is, as you can see here:

> Create a file named `kics.config` and place it in the working directory of your project repository, most of the time will be the root directory.

I couldn't think other place on docs to add about this change, do you have any suggestion? 😅

Reply:
Hello, @lipeavelar!

Sorry for the misunderstanding, but we would like to keep "the old verification" plus your suggestion. The idea is to keep both approaches and look at them. Can you change it, please?

I think the `docs/configuration-file.md` is the perfect location 😊 Thank you!